### PR TITLE
feat(artwork): Fix Duplicate Station Image

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
Solves https://github.com/endless-sky/endless-sky/issues/9356.

Removes station15, a duplicate of station7, and adjusts copyright/the use accordingly.

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/380

### CODE REVIEW

**Comment by ziproot** [comment]
The [picture you chose](https://unsplash.com/photos/a-dimly-lit-room-with-a-bunch-of-bars-6Yia_w7hD6s) is not in the public domain. The unsplash license prohibits selling, so the picture uses a non-free license.

**Comment by bene-dictator** [comment]
We're not selling the picture?

Besides if what you were saying was true we'd need to find a lot of new pictures very quickly since they're almost all from Unsplash.

**Comment by Saugia** [comment]
As bene says, we are not selling. There are a lot of images in our files that are through unsplash and unsplash's license. If it were the case, then we'd be replacing half of our landscape images, including ones that have been in the game since early iterations.

**Comment by ziproot** [comment]
Both of you used the strawman fallacy and argumentum ad consequentiam. 

Argumentum ad consequentiam: Whether or not the pictures are under a free license is not dependent on how easy it would be to use pictures with a free license in the video game "Endless Sky."

Strawman: I didn't say anything about using or not using this picture. You seemed to imply that I was advocating for abandoning this picture. I said that it uses a non-free license, which is true, but it was based on a misconception on my part for which I am sorry. Furthermore, not all pictures uploaded to Unsplash were always subject to that license, which you seem to think. Before June 2017, all pictures uploaded to Unsplash were CC0. Unfortunately I did say something incorrect: I said the unsplash license prohibits selling, because I was looking at the top part:
What is not permitted
    **Photos cannot be sold without significant modification.** (emphasis mine)
    Compiling photos from Unsplash to replicate a similar or competing service.

Unfortunately, I was wrong even this way because I said "prohibits" when I really should have said "restricts." I'm sorry. As the paragraph below it doesn't mention anything about selling, I was confused, until I realized that that line comes from their terms and conditions which they updated in June 2017 to restrict selling.

The paragraph only mentions the second one, and the first restriction is under their terms and conditions. I didn't realize that at first. I'm sorry. 

As for replacing every image in the game: none of them say "unsplash license." I assumed that was because they were published and added to the game before June 2017, when they were released under the CC0 license. This picture was published in 2023 and therefore was not released to the public domain. If they were released after and developers didn't actually check that the pictures were really public domain, then these need to be updated to say "unsplash license" instead. 

I am not done commenting but I want to at least clear a few things up.

**Comment by Saugia** [comment]
If the concern is that Unsplash images are not properly listed in the copyright files using "Unsplash license", then that can be changed to be the case. What you presented came off across rather "do not use this picture", which is what our responses were to.

**Comment by ziproot** [comment]
OK, as I said in the other PR, I sorted the PRs into public domain and not. As for this one, my first comment was badly worded. I'm sorry. 

It looks like a large portion of the images from unsplash in game were added as "public domain" with the assumption that if it was on unsplash.com, it was automatically public domain, well after unsplash had changed their terms. It could be that they actually were public domain due to being uploaded earlier, but I have no way of knowing.

I really think that from now on, the source of the images should be linked and PR authors and reviewers should take care to verify that the images they find actually have the license they claim. Unsplash is not the only website that started out as CC0 and changed, though they seem to be the only one to hide the fact that old images are CC0. I will probably make a PR fixing the unsplash ones soon.

Also, it looks like the "unsplash license" was at some point CC0: https://web.archive.org/web/20160903121417/https://unsplash.com/license, so I can understand the assumption. Still, terms and conditions can change.

I'm sorry for the confusion.

File: copyright [hunk 1]
```diff
@@ -95,6 +95,12 @@ Copyright: Emily Mell <hasmidas@gmail.com>
 License: public-domain
 Comment: Based on public domain images taken from unsplash.com
 
+Files:
+ images/land/station15
+Copyright: Andrea De Santis on unsplash.com
+License: Unsplash License
+Comment: Rotated, cropped, and edited by Saugia.
```
Comment (ziproot) [inline copyright:102]:
```suggestion
Comment: Rotated, cropped, and edited by Saugia. This picture was taken from
unsplash.com after June 2017, so it is subject to the Unsplash License.
```

**Review by ziproot — APPROVED** [review]
This has the correct label, and I am not going to reject this because it uses the Unsplash License, as in that case, as mentioned by bene-dictator and Saugia, it would mean I would also have to reject a large portion of the land images and all of the skadenga scene images, or try to find the original source to see if its public domain. When I initially said it was non-free, that was based on Wikipedia's judgment, and I didn't realize that so many images were mistakenly labeled. It still could be non-free, however, as I said, that is a different question than whether or not to include this in the game. Until #9413 gets resolved, I think that including Unsplash images is fine, and the resolution of 9413 should take the fact that a lot of images are probably under the potentially non-free Unsplash License into consideration.

**Comment by Saugia** [comment]
I'm closing this PR as unsplash.com license isn't appropriately accepted by Debain, and going forward, we'll be revisiting what our license policy is for any future images (to which this one does indeed not fall under). For more information on this, please refer back to https://github.com/endless-sky/endless-sky/issues/9413 as we expand upon it.

As an alternative, I will resolve 9356 by removing the duplicate image and simply point the Korath Station to the original.

**Comment by Saugia** [comment]
Accident.

**Review by ziproot — APPROVED** [review]
That works too

**Comment by ziproot** [comment]
It looks like the duplicate image replaced one that was licensed as "Pixabay License," however, it was published to the site in January 2018 when all pictures uploaded to Pixabay were public domain: https://web.archive.org/web/20181015044007/https://pixabay.com/service/terms/. You could just replace it with https://pixabay.com/photos/step-industry-steel-3104846 and say public domain, explaining why.

File: copyright [hunk 2]
```diff
@@ -736,7 +735,7 @@ License: CC0
 Comment: Taken from https://pxhere.com/en/photo/1071635 and cropped.
 
 Files:
- images/land/station15*
+ images/land/station7*
 Copyright: N/A (CC0 Public Domain)
 License: CC0
 Comment: Taken from https://pxhere.com/en/photo/119196 and cropped.
```
Comment (ziproot) [inline copyright:741]:
```suggestion
 images/land/station7*
Copyright: N/A (CC0 Public Domain)
License: CC0
Comment: Taken from https://pxhere.com/en/photo/119196 and cropped.

Files:
 images/land/station15*
Copyright: N/A (CC0 Public Domain)
License: CC0
Comment: 
 Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. 
 According to Section 3  of the Pixabay Terms and Conditions, all images with a 
 "published date" earlier than January 9, 2019 have the CC0 license.
```

**Review by ziproot — CHANGES_REQUESTED** [review]
As said in my earlier comment, you can replace station15.jpg with the image uploaded there. These changes fix the copyright.

**Review by ziproot — APPROVED** [review]
Problem solved

File: copyright [hunk 3]
```diff
@@ -740,6 +740,15 @@ Copyright: N/A (CC0 Public Domain)
...
 
+Files:
+ images/land/station15*
+Copyright: N/A (CC0 Public Domain)
+License: CC0
+Comment: 
+ Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. 
+ According to Section 3 of the Pixabay Terms and Conditions, all images with a
```
Comment (ziproot) [inline copyright]: I forgot to remove the trailing whitespace from these lines.

File: copyright [hunk 4]
```diff
@@ -736,11 +735,17 @@ License: CC0
...
+ images/land/station7*
 Copyright: N/A (CC0 Public Domain)
 License: CC0
 Comment: Taken from https://pxhere.com/en/photo/119196 and cropped.
 
+Files:
+ images/land/station15*
+Copyright: N/A (CC0 Public Domain)
+License: CC0
+Comment: Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. According to Section 3 of the Pixabay Terms and Conditions, all images with a "published date" earlier than January 9, 2019 have the CC0 license.
```
Comment (warp-core) [inline copyright:747]: The page calls them "Terms of Service".
Should the last updated date be included, as well? It currently says April 4, 2023.
```suggestion
Comment: Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. According to Section 3 of the Pixabay Terms of Service, all images with a "published date" earlier than January 9, 2019 have the CC0 license.
```

File: copyright [hunk 5]
```diff
@@ -736,11 +735,19 @@ License: CC0
...
 License: CC0
 Comment: Taken from https://pxhere.com/en/photo/119196 and cropped.
 
+Files:
+ images/land/station15*
+Copyright: N/A (CC0 Public Domain)
+License: CC0
+Comment: Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped.
+According to Section 3 of the Pixabay Terms of Service, all images with a 
+"published date" earlier than January 9, 2019 have the CC0 license.
```
Comment (Amazinite) [inline copyright:749]:
```suggestion
Comment:
 Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped.
 According to Section 3 of the Pixabay Terms of Service, all images with a 
 "published date" earlier than January 9, 2019 have the CC0 license.
```

**Pull request merged**